operator/src/main/resources/Operator.properties

```diff
@@ -194,7 +194,7 @@ WLSDO-0009=The secret ''{0}'' must be specified if ''domainHomeSourceType'' is c
 WLSDO-0010=The secret ''{0}'' must be specified if ''domainHomeSourceType'' is configured as ''FromModel'' and spec.configuration.model.domainType is configured as ''JRF''.
 WLSDO-0011=Cluster ''{0}'' specified in the domain resource does not exist in the WebLogic domain home configuration.
 WLSDO-0012=Managed Server ''{0}'' specified in the domain resource does not exist in the WebLogic domain home configuration.
-
+WLSDO-0013=Istio is enabled and the domain resource specified to expose default channel in adminServices, this is not allowed. Please create a separeate network channel and then expose it
```

Comment:
Needs to also mention default-secure and default-admin. Collapse to one sentence.

Reply:
done